packages/converter/package.json

```diff
@@ -13,8 +13,7 @@
   ],
   "scripts": {
     "build": "tsc",
-    "pretest": "npm run build",
-    "test": "jest --coverage",
+    "test": "echo 'skip'",
```

Comment:
This entire package file will be removed in the next step. Keeping the test script to continue to run lerna test so that the core posttest script can run.